src/Gauge/BigTrafficWidget.cpp

```diff
@@ -39,7 +39,7 @@ class FlarmTrafficControl : public FlarmTrafficWindow {
 public:
   FlarmTrafficControl(const FlarmTrafficLook &look)
     :FlarmTrafficWindow(look, Layout::Scale(10),
-                        Layout::GetMinimumControlHeight() + Layout::Scale(2)) {}
+                        Layout::GetMinimumControlHeight() + Layout::Scale(10)) {}
```

Comment:
isn't this *increasing* the padding? or is that done because the second part of the PR is increasing the size of the circles?

Reply:
Yes increasing the padding. This part is just cosmetic. As in default the buttons at the top overlap the circle in landscape mode.

Reply:
![Screenshot_20240314-091425](https://github.com/XCSoar/XCSoar/assets/407371/376d9a47-80e5-4826-81db-4325413cccf8)

Reply:
I see, thanks! :)